src/main/java/servlets/VendorServlet.java

```diff
@@ -33,12 +45,75 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     JsonConverter converter = new JsonConverter();
     String configMap = converter.getConfigMap();
-    response.setContentType("text/html;");
+    response.setContentType(CONTENT_TYPE);
     response.getWriter().println(configMap);
   }
 
-  @Override
-  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // DELETE method --> deletes vendor configuration directory
+  /** Delete the desired Vendor from the fileset and spreadsheets if posible.*/
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {  
+    String vendorID = request.getParameter(VENDOR_ID_PARAM);
+    String deleteStatus = "success";
+
+    // Delete the file from the filesystem and update the sheets.
+    try {
+      deleteVendorFile(vendorID);
+      updateSheets(request);    
+    } catch (FileNotFoundException e) {
+      deleteStatus = "Could not execute, an FileNotFoundException was thrown.";
```

Comment:
I don't think that saying which type of exception was thrown is that useful to a user.

Maybe say something generic like "Failed to delete file" or something more specific like "The specified vendor was not found."

Reply:
Will do